main.nf

```diff
@@ -586,7 +592,7 @@ process markDuplicates {
 
     script:
     """
-    java -Xmx2g -jar \$PICARD_HOME/picard.jar MarkDuplicates \\
+    java -Xmx${params.java_mem} -jar \$PICARD_HOME/picard.jar MarkDuplicates \\
```

Comment:
Probably - I was thinking that specifying the exact amount of memory could break things if there is any other overhead, so specifying 90% of available memory would be safer. But it's probably not needed.